Content.Server/ParticleAccelerator/EntitySystems/ParticleAcceleratorSystem.Parts.cs

```diff
@@ -48,12 +48,12 @@ public void RescanParts(EntityUid uid, ICommonSession? user = null, ParticleAcce
             return;
 
         var gridUid = xform.GridUid;
-        if (gridUid == null || gridUid != xform.ParentUid || !_mapManager.TryGetGrid(gridUid, out var grid))
+        if (gridUid == null || gridUid != xform.ParentUid || !TryComp<MapGridComponent>(gridUid, out var grid))
             return;
 
         // Find fuel chamber first by scanning cardinals.
         var fuelQuery = GetEntityQuery<ParticleAcceleratorFuelChamberComponent>();
-        foreach (var adjacent in grid.GetCardinalNeighborCells(xform.Coordinates))
+        foreach (var adjacent in _mapSystem.GetCardinalNeighborCells(gridUid!.Value, grid, xform.Coordinates))
```

Comment:
```suggestion
        foreach (var adjacent in _mapSystem.GetCardinalNeighborCells(gridUid.Value, grid, xform.Coordinates))
```

Null suppression is unnecessary here as it's already checked above.